packages/pg-v5/commands/credentials/rotate.js

```diff
@@ -16,7 +16,9 @@ async function run(context, heroku) {
     throw new Error('cannot pass both --all and --name')
   }
 
-  if (util.essentialPlan(db) && cred !== 'default') {
+  if (util.essentialNumPlan(db)) {
+    throw new Error('You can’t perform this operation on Essential-tier databases.')
+  } else if (util.legacyEssentialPlan(db) && cred !== 'default') {
```

Comment:
can we combine these?
```suggestion
  if ((util.essentialNumPlan(db) || util.legacyEssentialPlan(db)) && cred !== 'default') {
```

Reply:
yes combined those ^